Microsoft.Toolkit.Uwp.SampleApp/SamplePages/SystemInformation/SystemInformationCode.bind

```diff
@@ -37,6 +37,9 @@
         // To get the first version installed
         public PackageVersion FirstVersionInstalled => SystemInformation.Instance.FirstVersionInstalled;
 
+        // To get the previous version installed
+        public string PreviousVersionInstalled => SystemInformation.Instance.PreviousVersionInstalled.ToFormattedString();
```

Comment:
Why are these two not matching?
They should return the same type and have the same logic.
If we're returning a `string` here, we should also update the sample code for `FirstVersionInstalled` then.

Reply:
@Panda-Sharp ⬆️

Reply:
@Sergio0694 fixed, my bad I copied and pasted from the sample app page instead of this one.